whylog_vim/gui/windows.py

```diff
@@ -0,0 +1,48 @@
+from whylog_vim.gui.exceptions import CannotFindWindowId
+from whylog_vim.gui.files_manager import FilesManager
+from whylog_vim.gui.vim_ui_wrapper import VimUIWrapper
+
+
+class WindowContext(object):
+    def __enter__(self):
+        VimUIWrapper.set_modifiable()
+        return VimUIWrapper.get_buffor()
+
+    def __exit__(self, type_, value, traceback):
+        VimUIWrapper.set_nomodifible()
+
+
+class Window(object):
+    def __init__(self, name, content=None, modifiable=False, splited_window_size=None):
```

Comment:
Function interface suggests that the `content` parameter is optional, but it really isn't. Please remove the default value.